src/sentry_random.c

```diff
@@ -1,24 +1,25 @@
 #include "sentry_boot.h"
 
-#include "sentry_random.h"
-#include "sentry_utils.h"
+#ifndef SENTRY_PLATFORM_NX
```

Comment:
As described [on discord](https://discord.com/channels/621778831602221064/1343728776017739829/1345021291748393012), if you exclude the entire translation unit, it may be better to do it in CMake.

Reply:
OK, excluded in cmake now